web/pgadmin/tools/sqleditor/utils/query_tool_preferences.py

```diff
@@ -366,13 +389,31 @@ def register_query_tool_preferences(self):
 
     self.preference.register(
         'keyboard_shortcuts',
-        'execute_query',
+        'execute_script',
```

Comment:
You changed `execute_query` to `execute_script` and used `execute_query` for new code. This is dangerous if user has customised the execute_query shortcut as it will get assigned to new button. You'll need to add a migration if you want to change this.

Reply:
I have kept existing execute_query to execute_script, and the new execute button to execute_cursor.

Reply:
read my above comment.